internal/app/machined/pkg/adapters/cluster/identity_test.go

```diff
@@ -25,7 +25,7 @@ func TestIdentityGenerate(t *testing.T) {
 	length := len(spec1.NodeID)
 
 	assert.GreaterOrEqual(t, length, 43)
-	assert.LessOrEqual(t, length, 44)
+	assert.LessOrEqual(t, length, 45)
```

Comment:
what was this change?

Reply:
just fixing flaky tests, this encoding has variable length in characters